cmd/connectivity-registry/main.go

```diff
@@ -22,14 +22,21 @@ import (
 
 func main() {
 	var (
-		tlsCertPath string
-		tlsKeyPath  string
-		port        int
+		tlsCertPath                                   string
+		tlsKeyPath                                    string
+		port                                          int
+		orphanImportedServiceRecordDeleteDelaySeconds int
 	)
 
 	flag.StringVar(&tlsCertPath, "tls-cert", "", "the path to the server TLS certificate")
 	flag.StringVar(&tlsKeyPath, "tls-key", "", "the path to the server TLS key")
 	flag.IntVar(&port, "port", 8000, "the serving port for the hamlet server")
+	flag.IntVar(
+		&orphanImportedServiceRecordDeleteDelaySeconds,
+		"orphan-imported-service-record-delete-delay-seconds",
```

Comment:
Whoa, this is quite long, but I'm not sure if the verbosity really helps make this flag clearer.

Reply:
Open to suggestions!

Reply:
I wonder if something like "imported-service-record-ttl" would be clearer

Reply:
We threw around using TTL, but decided against it because flag does not specify a TTL of service records. I was afraid it would be misleading if we used TTL in the name.

Reply:
how about `stale-import-cleanup-delay` as a [DurationVar](https://golang.org/pkg/flag/#DurationVar)

Reply:
or ignore me and just merge it